src/js/dom.js

```diff
@@ -0,0 +1,128 @@
+
+function addContent() {
+  const container = document.getElementById('container');
+
+
+  const addBook = document.createElement('div');
+  addBook.classList.add('addBook');
+  container.setAttribute('style', 'text-align: center; background-color: gray ');
+  container.appendChild(addBook);
+
+
+  const form = document.createElement('form');
+  form.innerHTML = '<h3>Add a Book</h3>';
+  form.setAttribute('style', 'margin: 1em');
+  form.classList.add('form-group');
+  container.appendChild(form);
+
+
+  const titleDiv = document.createElement('div');
+  titleDiv.setAttribute('style', 'margin: 0.5em');
+  const title = document.createElement('input');
+  title.setAttribute('id', 'title');
+  title.setAttribute('type', 'text');
+  title.setAttribute('placeholder', 'Title');
+  title.classList.add('form-control');
+  titleDiv.appendChild(title);
+  form.appendChild(titleDiv);
+
+
+  const authorDiv = document.createElement('div');
+  authorDiv.setAttribute('style', 'margin: 0.5em');
+  const author = document.createElement('input');
+  author.setAttribute('id', 'author');
+  author.setAttribute('type', 'text');
+  author.setAttribute('placeholder', 'Author');
+  author.classList.add('form-control');
+  authorDiv.appendChild(author);
+  form.appendChild(authorDiv);
+
+
+  const pagesDiv = document.createElement('div');
+  pagesDiv.setAttribute('style', 'margin: 0.5em');
+  const pages = document.createElement('input');
+  pages.setAttribute('id', 'pages');
+  pages.setAttribute('type', 'number');
+  pages.setAttribute('placeholder', 'Pages');
+  pages.classList.add('form-control');
+  pagesDiv.appendChild(pages);
+  form.appendChild(pagesDiv);
+
+
+  const submitDiv = document.createElement('div');
+  submitDiv.setAttribute('style', 'margin: 0.5em');
+  const submit = document.createElement('button');
+  submit.setAttribute('id', 'submit');
+  submit.setAttribute('type', 'submit');
+  submit.innerHTML = 'New Book';
+  submit.classList.add('btn');
+  submit.classList.add('btn-primary');
+  submitDiv.appendChild(submit);
+  form.appendChild(submitDiv);
+
+
+  return addBook;
+}
+
+
+function createTable(book, i) {
+  const container = document.getElementById('container');
+
+  const table = document.createElement('table');
+  table.classList.add('table');
+  table.setAttribute('style', 'margin: 0.5em; width: 100%; display: flex; align-items: center');
+  table.setAttribute('id', 'id');
+  table.classList.add('container');
+  table.classList.add('m-2');
+  table.id = book.id++;
```

Comment:
- [ ] Here you can remove the ++ and Eslint it will stop complaining. We can instead use:
`table.id = book.id += 1;`
Since you disabled the `no-plusplus` Eslint rule I'm assuming you had this problem